streamalert/shared/lookup_tables/utils.py

```diff
@@ -0,0 +1,70 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from streamalert.shared.lookup_tables.drivers import PersistenceDriver
+
+
+# pylint: disable=protected-access
```

Comment:
`# pylint: disable=...` comments apply to the entire _scope_ where they're added. please only apply this where you need to (preferably on the exact line only)